server/publications/settings/index.js

```diff
@@ -4,13 +4,17 @@ import { Settings } from '../../../app/models/server';
 import { Notifications } from '../../../app/notifications/server';
 import { hasPermission, hasAtLeastOnePermission } from '../../../app/authorization/server';
 import { getSettingPermissionId } from '../../../app/authorization/lib';
+import { SettingsEvents } from '../../../app/settings/server/functions/settings';
 
 Meteor.methods({
 	'public-settings/get'(updatedAt) {
+		const mapFn = ({ _id, value, editor }) => ({ _id, value, editor });
```

Comment:
Do not need to remove the enterprise info, the UI will need to know which are enterprise to show a badge, even the modules are useful so we can show which one is required to use the setting.